code/drasil-code/lib/Language/Drasil/Code/Imperative/Descriptions.hs

```diff
@@ -85,7 +85,7 @@ inputFormatDesc = do
   g <- get
   let ifDesc False = ""
       ifDesc _ = "the function for reading inputs"
-  return $ ifDesc $ "get_input" `elem` defList g
+  return $ ifDesc $ genICFuncName GetInput `elem` defList g
```

Comment:
same here - it feels like `defList` should have its type "improved".

Reply:
The type of `defList` type has been changed in a986e0cf111cee60ab034fb92462537589e8bdcd but feels a bit hacky with my use of `listStrIC` and `swap` . Please provide some feedback on improvements and if they are needed in this PR.